package.json

```diff
@@ -22,8 +21,7 @@
     "nodemailer-express-handlebars": "^3.0.0",
     "objection": "^1.1.9",
     "pg": "^7.4.3",
-    "popsicle": "^9.2.0",
-    "request": "^2.86.0"
+    "popsicle": "^9.2.0"
```

Comment:
Can we get rid of popsicle from package.json now?

Reply:
oh, yeah sorry I meant to do that.